samples/XAMLCatalog/XAMLCatalog/UIKitControls/UIButtonWithControlsViewController.m

```diff
@@ -64,9 +64,35 @@ - (UIControlState)getControlState:(NSString*)stateString {
     return state;
 }
 
+- (UIEdgeInsets)getUIEdgeInsetsFromString:(NSString*)edgeInsetsString {
+    UIEdgeInsets ret = UIEdgeInsetsZero;
+
+    if ([edgeInsetsString length] > 0) {
+        // Strip off curly braces and use the delimiter to separate components
```

Comment:
>  // Strip off curly braces and use the delimiter to separate components [](start = 6, length = 72)

How does the user know what format to use here?  Tons of comments would be helpful.